### PR TITLE
[dv-processing] 1.7.8 update

### PR DESCRIPTION
**Pull request by llongi**
Update the dv-processing package to its latest 1.7.8 release.

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by llongi** [comment]
@microsoft-github-policy-service agree company="iniVation AG"

File: ports/dv-processing/portfile.cmake [hunk 1]
```diff
@@ -2,27 +2,27 @@ vcpkg_from_gitlab(
...
 
 file(GLOB CMAKEMOD_FILES ${CMAKEMOD_SOURCE_PATH}/*)
-file(COPY ${CMAKEMOD_FILES} DESTINATION ${SOURCE_PATH}/cmakemod)
+file(COPY ${CMAKEMOD_FILES} DESTINATION ${SOURCE_PATH}/cmake/modules)
 
 vcpkg_cmake_configure(
         SOURCE_PATH ${SOURCE_PATH}
         OPTIONS
-	        -DENABLE_TESTS=OFF
+		-DENABLE_TESTS=OFF
```
Comment (Adela0814) [inline ports/dv-processing/portfile.cmake:25]: Please use four spaces instead of tabs, thanks.

File: ports/dv-processing/portfile.cmake [hunk 2]
```diff
@@ -2,27 +2,27 @@ vcpkg_from_gitlab(
...
-        SHA512 4fe9cc5099ab8b41c982df45cbf9a000b2cb1f1c6ed536685943a60520cff49e262ec43af8187177c50a0df2dfca57e7861bf2e7d07834fc16e85c30eb9a9edb
-        HEAD_REF a4d7eccfdc5f83e399786a77df79b178b762858b
+        REF ec53dae89f6b037e9e640af5340d7bf67d84d278
+        SHA512 e7907b1be9d85b02e1a1703cf001765119a7d07b1873148a0fbfe6945c519d85b1f9bc66b24f90d88759c2b32965304e1639f2ff136448be64fc88f81a0d4c2d
+        HEAD_REF ec53dae89f6b037e9e640af5340d7bf67d84d278
 )
 
 file(GLOB CMAKEMOD_FILES ${CMAKEMOD_SOURCE_PATH}/*)
-file(COPY ${CMAKEMOD_FILES} DESTINATION ${SOURCE_PATH}/cmakemod)
+file(COPY ${CMAKEMOD_FILES} DESTINATION ${SOURCE_PATH}/cmake/modules)
```
Comment (Adela0814) [inline ports/dv-processing/portfile.cmake:20]:
```suggestion
file(COPY ${CMAKEMOD_FILES} DESTINATION "${SOURCE_PATH}/cmake/modules")

```

**Review by Adela0814 — CHANGES_REQUESTED** [review]

File: ports/dv-processing/portfile.cmake [hunk 3]
```diff
@@ -2,27 +2,27 @@ vcpkg_from_gitlab(
...
         REPO dv/cmakemod
-        REF a4d7eccfdc5f83e399786a77df79b178b762858b
-        SHA512 4fe9cc5099ab8b41c982df45cbf9a000b2cb1f1c6ed536685943a60520cff49e262ec43af8187177c50a0df2dfca57e7861bf2e7d07834fc16e85c30eb9a9edb
-        HEAD_REF a4d7eccfdc5f83e399786a77df79b178b762858b
+        REF ec53dae89f6b037e9e640af5340d7bf67d84d278
+        SHA512 e7907b1be9d85b02e1a1703cf001765119a7d07b1873148a0fbfe6945c519d85b1f9bc66b24f90d88759c2b32965304e1639f2ff136448be64fc88f81a0d4c2d
+        HEAD_REF ec53dae89f6b037e9e640af5340d7bf67d84d278
 )
 
 file(GLOB CMAKEMOD_FILES ${CMAKEMOD_SOURCE_PATH}/*)
```
Comment (Adela0814) [inline ports/dv-processing/portfile.cmake:19]:
```suggestion
file(GLOB CMAKEMOD_FILES "${CMAKEMOD_SOURCE_PATH}/*")

```

**Review by Adela0814 — CHANGES_REQUESTED** [review]

File: ports/dv-processing/portfile.cmake (hunk 1 above)
Comment (llongi) [reply ports/dv-processing/portfile.cmake]: Done, I've updated the whole file to use 4-spaces indentation, before it was a mix of 8-spaces and tabs.

**Review by Adela0814 — APPROVED** [review]

**Pull request merged**